rmp/src/value.rs

```diff
@@ -481,31 +481,51 @@ impl From<usize> for Value {
 
 impl From<i8> for Value {
     fn from(v: i8) -> Value {
-        Value::Integer(Integer::I64(From::from(v)))
```

Comment:
I changed it due to the comment on https://github.com/3Hren/msgpack-rust/blob/master/rmp/src/value.rs#L14, and makes `Value::from` the same as `Deserialize`. I don't feel strongly either way, but it does simplify the tests.